ras_party/controllers/account_controller.py

```diff
@@ -291,6 +291,17 @@ def request_password_change(payload, session):
     return {'response': "Ok"}
 
 
+@with_db_session
+def change_respondent_account_status(payload, party_id, session):
+
+    status = payload['status_change']
+
+    respondent = query_respondent_by_party_uuid(party_id, session)
+    if not respondent:
+        raise RasError("Unable to find respondent account", status=404)
+    respondent.status = status
+
```

Comment:
Are you wanting to return the respondent back from this function? if not the codacy error may need addressing.